mapper/.devcontainer.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Let's change the directory name to `iso20022_to_switmt` to make it consistent with the other repo

Reply:
Ack